### DISCUSSION TITLE
Can the tiddler created on clipping be automatically opened in the story river

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#80] [Q&A] [answered]
Currently whenever we clip text, we have to go to recent tiddler and open the clipped tiddler from there. Can the tiddler created on clipping be automatically opened in the story river after the clipping.

**Comment by buggyj** [comment, marked as answer]
Add 'open' to the modes:

![image](https://user-images.githubusercontent.com/2855795/156817662-34ee5aa2-b5c2-4932-b93e-246d7205dd02.png)
